internal/install/ux/spinner.go

```diff
@@ -52,10 +50,12 @@ func (s *Spinner) Stop() {
 	}
 }
 
-func (s *Spinner) Fail(recipe types.Recipe) {
-	s.FinalMSG = crossmark
+func (s *Spinner) Fail(msg string) {
+	s.FinalMSG = indentation + crossmark
+	s.Suffix = s.Suffix + "failed."
 }
 
-func (s *Spinner) Success(recipe types.Recipe) {
-	s.FinalMSG = checkmark
+func (s *Spinner) Success(msg string) {
+	s.FinalMSG = indentation + checkmark
+	s.Suffix = s.Suffix + "success."
```

Comment:
Will this cause `s.Suffix` to miss a space between it and 'success'?

Reply:
The intended text is "...success.", so no space is required.